### ISSUE TITLE
Python + HTTPX

### ISSUE THREAD

**Issue by techux** [curlconverter/curlconverter#706]
https://www.python-httpx.org/

**Comment by 0x24a** [comment]
> just add "import httpx as requests" at the top to implement this

Not exactly. curlconverter assumes `requests` doesn't support HTTP/2 and shows a warning.
